docs/src/test/java/sample/javadsl/FetchMetadata.java

```diff
@@ -0,0 +1,43 @@
+package sample.javadsl;
+
+// #metadata
+        import akka.actor.ActorRef;
+        import akka.kafka.KafkaConsumerActor;
+        import akka.kafka.Metadata;
+        import akka.pattern.PatternsCS;
+        import akka.util.Timeout;
+        import org.apache.kafka.common.TopicPartition;
+
+        import java.util.concurrent.CompletionStage;
+        import java.util.concurrent.TimeUnit;
```

Comment:
Ooh, what happened here to the formatting?

Reply:
This indentation makes the code snippet look nicer.